benchmark/scanner/credsweeper.py

```diff
@@ -26,8 +27,8 @@ def init_scanner(self) -> None:
     def run_scanner(self) -> None:
         self.init_scanner()
         subprocess.call([
-            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--ml_validation",
-            "--save-json", self.output_dir
+            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--save-json",
+            "--jobs", "4", self.output_dir
```

Comment:
Probably a wrong order
Should be `self.output_dir` should be wight after `--save-json`

Reply:
Oh.. I'll fix them.

Reply:
Fixed it with [6e284c2](https://github.com/Samsung/CredData/pull/18/commits/6e284c24ba95e275207489c18ef7ecda44f0f3e2).